src/EspressoNitroTEEVerifier.sol

```diff
@@ -33,20 +37,20 @@ contract EspressoNitroTEEVerifier is NitroValidator, IEspressoNitroTEEVerifier,
         Ownable()
     {
         _certManager = certManager;
-        registeredEnclaveHash[enclaveHash] = true;
+        registeredBatchPosterEnclaveHashes[enclaveHash] = true; // TODO: modify this constructor after some review on this.
         _transferOwnership(msg.sender);
     }
 
     /**
-     * @notice This function registers a new signer by verifying an attestation from the AWS Nitro Enclave (TEE)
+     * @notice This function registers a new Caff Node by verifying an attestation from the AWS Nitro Enclave (TEE)
      * @param attestation The attestation from the AWS Nitro Enclave (TEE)
      * @param signature The cryptographic signature over the COSESign1 payload (extracted from the attestation)
      */
-    function registerSigner(bytes calldata attestation, bytes calldata signature) external {
+    function registerCaffNode(bytes calldata attestation, bytes calldata signature) external {
```

Comment:
i recognize that this needs to be permissionless but i wonder if we need some controls (that can be decided on by governance). Some options are rate limiting or pausability

Reply:
I agree with @alysiahuggins  here that we should look this but can do it in a follow up PR!